workspaces/applications.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Applications"
+description:
+  Learn how to access web applications running on your remote workspace.
+state: beta
+---
+
+You can connect to web applications installed on your workspace using the
+applications specification file located at `/coder/apps/config.yaml` of the
+workspace filesystem.
+
+![Application Launcher](../assets/workspaces/applications.png)
+
+## Application specification file
+
+The application specification file allows you to define what Coder needs to
+launch the application. Within the file, you can specify the following for each
+application:
+
+```yaml
+# /coder/apps/config.yaml
+
+apps:
+  # Name of application in launcher
+  - name: projector
```

Comment:
what can be in the name?

Reply:
I don't believe we have any restrictions on the name. Can have space, emojis, etc.